.github/workflows/command-backport.yml

```diff
@@ -0,0 +1,42 @@
+name: Backport into stable
+
+on:
+  pull_request_target:
+    types: [closed]
+  issue_comment:
+    types: [created]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+jobs:
+  backport:
+    name: Backport pull request
+    runs-on: ubuntu-latest
+
+    # Excluding CI action user id `97796249`:
+    if: >
+      (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.merged
+      ) || (
+        github.event_name == 'issue_comment' &&
+        github.event.issue.pull_request &&
+        github.event.comment.user.id != 97796249 &&
```

Comment:
I think this should be limited to contributors/members since it creates a PR from the bot, which I assume runs all CI without needing approval. The tag route is already limited since only members can add tags. Otherwise somebody could comment `/backport` on many merged PRs and spam the CI

Reply:
Ah right. I thought that org membership is not needed since it just opens a MR, but yes it probably has CI approval...

Will add, thanks.

Reply:
I removed the `/backport` trigger and only use labels now.

The whole thing should be safe, since it only runs on merge requests that got merged into master.  
If someone manages to merge malicious CI code into master then it is already a problem.

Reply:
I just meant for the wasted work on the CI, but removing the comment sounds good.